CMakeLists.txt

```diff
@@ -14,9 +14,9 @@ target_compile_features(despacer PRIVATE "c_std_99")
 message(STATUS "CMAKE_SYSTEM_PROCESSOR = ${CMAKE_SYSTEM_PROCESSOR}")
 if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64" OR  "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL AMD64)
   if(MSVC)
-    target_compile_options(despacer PRIVATE /arch:native)
+    target_compile_options(despacer PRIVATE /arch:native /O2)
   else()
-    target_compile_options(despacer PRIVATE -march=native)
+    target_compile_options(despacer PRIVATE -march=native -O3)
```

Comment:
Thanks, perhaps we could add instructions for this to the top-level README instead of editing CMakeLists.txt. Currently people who follow these instructions get a build with no optimizations.

Reply:
"Currently people who follow these instructions get a build with no optimizations."

But that's the default in CMake, right? When invoking cmake, by default you get no optimization. Correct?